tests/test_data/test_pipelines/test_transform/test_transform.py

```diff
@@ -10,6 +10,7 @@
 
 from mmdet.core.evaluation.bbox_overlaps import bbox_overlaps
 from mmdet.datasets.builder import PIPELINES
+from .utils import create_random_bboxes
```

Comment:
If relative import is adopted, is it impossible to run unit tests in the current folder?

Reply:
Import is okay because there is a __init__.py here.
However, some config and image paths use relative paths. So it still needs to run pytest at the root folder of the project.

Reply:
That's a bit troublesome, especially for users who want to use IDE directly. In this case, it is acceptable.